ios/FluentUI/Vnext/Core/UIKit+SwiftUI_interoperability.swift

```diff
@@ -23,6 +24,9 @@ struct UIViewAdapter: UIViewRepresentable {
     }
 
     func updateUIView(_ view: UIView, context: Context) {
+        if let tintColor = tintColor {
```

Comment:
I think a much better alternative would be to provide a closure handler to the client where they can update the view themselves. 

```swift
    var updateView: (UIView) -> UIView
```

this way you won't need extension for every additional properties that client wants to modify.